ci/test.sh

```diff
@@ -89,6 +101,6 @@ fi
 
 markEndOfBlock "$0"
 
-if [ $EDITMODE_TEST_RESULT -ne 0 ] || [ $PLAYMODE_TEST_RESULT -ne 0 ] || [ $CODE_GENERATOR_TEST_RESULT -ne 0 ] || [ $TOOLS_TEST_RESULT -ne 0 ]; then
+if [ $EDITMODE_TEST_RESULT -ne 0 ] || [ $PLAYMODE_TEST_RESULT -ne 0 ] || [ $CODE_GENERATOR_TEST_RESULT -ne 0 ] || [ $CODE_GENERATOR_E2E_TEST_RESULT -ne 0 ] || [ $TOOLS_TEST_RESULT -ne 0 ]; then
```

Comment:
Can you split this into multiple lines? (I'm not sure of bash syntax gotchas)

Reply:
Done